.github/dependabot.yaml

```diff
@@ -7,6 +7,10 @@ updates:
       interval: weekly
     commit-message:
       prefix: chore
+    groups:
+      go-modules:
```

Comment:
PR title will be `chore: Bump the go-modules group across 1 directory with X updates`